.github/workflows/ci-docker-hotrod.yml

```diff
@@ -18,6 +18,15 @@ permissions:
 jobs:
   hotrod:
     runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        mode:
+        - name: v1
+          binary: all-in-one
+        - name: v2
+          binary: jaeger
```

Comment:
please keep only the version in the matrix, and assign binary name inside the script